src/sentry/api/endpoints/group_participants.py

```diff
@@ -1,15 +1,18 @@
+from __future__ import annotations
+
 from rest_framework.request import Request
 from rest_framework.response import Response
 
 from sentry.api.base import region_silo_endpoint
 from sentry.api.bases import GroupEndpoint
-from sentry.api.serializers import serialize
-from sentry.models import GroupSubscriptionManager
+from sentry.services.hybrid_cloud.user import user_service
 
 
 @region_silo_endpoint
 class GroupParticipantsEndpoint(GroupEndpoint):
-    def get(self, request: Request, group) -> Response:
-        participants = GroupSubscriptionManager.get_participating_users(group)
+    def get(self, request: Request, group, organization_slug: str | None = None) -> Response:
+        participants = user_service.get_participating_users(group)
 
-        return Response(serialize(participants, request.user))
+        return Response(
+            user_service.serialize_users([participant.id for participant in participants])
```

Comment:
Once `user_service.get_participating_users` returns `APIUser` objects, you should be able to remove the call to `user_service.serialize_users`. That's potentially significant because calling two HC service methods in a row could eventually mean two high-latency RPCs. So it's worth thinking now about whether it's possible to do a whole operation in a single round trip.

Reply:
do we have a standard way to serialize `APIUser`? currently i get `TypeError: Object of type APIUser is not JSON serializable` when i just have `return Response(participants)` here after making the change you suggested above

Reply:
the conclusion to this is to not return a list of serialized APIUsers but a serialized list of Users